keyvi/tests/keyvi/dictionary/dictionary_compiler_test.cpp

```diff
@@ -345,6 +345,19 @@ BOOST_AUTO_TEST_CASE_TEMPLATE(DeleteUnsupported, SorterT, sorter_types) {
   BOOST_CHECK_THROW(compiler.Delete("aa"), compiler_exception);
 }
 
+BOOST_AUTO_TEST_CASE_TEMPLATE(MultipleKeyUpdateAndCompile, SorterT, sorter_types) {
+  keyvi::util::parameters_t params = {{"a", "1"}};
+
+  keyvi::dictionary::DictionaryCompiler<fsa::internal::SparseArrayPersistence<uint16_t>, fsa::internal::JsonValueStore,
+                                        SorterT>
+      compiler(params);
```

Comment:
you should be able to create the compiler without parameters.

however, we are used to decrease the memory limit as the default is quite high for unit tests.